ci/ray_ci/doc/test_api.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import pytest
+
+from ci.ray_ci.doc.api import (
+    API,
+    AnnotationType,
+    CodeType,
+    SPHINX_AUTOCLASS_HEADER,
+    SPHINX_AUTOSUMMARY_HEADER,
+)
+
+
+def test_from_autosummary():
+    test_harness = [
```

Comment:
nit: maybe just call it `test_data` ? or `test_cases`?

harness feels a bit too big of a word..